sql/moz-fx-data-shared-prod/telemetry_derived/firefox_use_counters_v1/query.sql

```diff
@@ -0,0 +1,16623 @@
+    SUM(
+      metrics.counter.use_counter_page_window_cssmathnegate
+    ) AS use_counter_page_window_cssmathnegate,
+    SUM(
+      metrics.counter.use_counter_page_window_cssmathproduct
+    ) AS use_counter_page_window_cssmathproduct,
+    SUM(metrics.counter.use_counter_page_window_cssmathsum) AS use_counter_page_window_cssmathsum,
+    SUM(
+      metrics.counter.use_counter_page_window_cssmathvalue
+    ) AS use_counter_page_window_cssmathvalue,
+    SUM(
+      metrics.counter.use_counter_page_window_cssmatrixcomponent
+    ) AS use_counter_page_window_cssmatrixcomponent,
+    SUM(
+      metrics.counter.use_counter_page_window_cssnumericarray
+    ) AS use_counter_page_window_cssnumericarray,
+    SUM(
+      metrics.counter.use_counter_page_window_cssnumericvalue
+    ) AS use_counter_page_window_cssnumericvalue,
+    SUM(
+      metrics.counter.use_counter_page_window_cssperspective
+    ) AS use_counter_page_window_cssperspective,
+    SUM(
+      metrics.counter.use_counter_page_window_csspositionvalue
+    ) AS use_counter_page_window_csspositionvalue,
+    SUM(
+      metrics.counter.use_counter_page_window_csspropertyrule
+    ) AS use_counter_page_window_csspropertyrule,
+    SUM(metrics.counter.use_counter_page_window_cssrotate) AS use_counter_page_window_cssrotate,
+    SUM(metrics.counter.use_counter_page_window_cssscale) AS use_counter_page_window_cssscale,
+    SUM(metrics.counter.use_counter_page_window_cssskew) AS use_counter_page_window_cssskew,
+    SUM(metrics.counter.use_counter_page_window_cssskewx) AS use_counter_page_window_cssskewx,
+    SUM(metrics.counter.use_counter_page_window_cssskewy) AS use_counter_page_window_cssskewy,
+    SUM(
+      metrics.counter.use_counter_page_window_cssstylevalue
+    ) AS use_counter_page_window_cssstylevalue,
+    SUM(
+      metrics.counter.use_counter_page_window_csstransformcomponent
+    ) AS use_counter_page_window_csstransformcomponent,
+    SUM(
+      metrics.counter.use_counter_page_window_csstransformvalue
+    ) AS use_counter_page_window_csstransformvalue,
+    SUM(
+      metrics.counter.use_counter_page_window_csstranslate
+    ) AS use_counter_page_window_csstranslate,
+    SUM(
+      metrics.counter.use_counter_page_window_cssunitvalue
+    ) AS use_counter_page_window_cssunitvalue,
+    SUM(
+      metrics.counter.use_counter_page_window_cssunparsedvalue
+    ) AS use_counter_page_window_cssunparsedvalue,
+    SUM(
+      metrics.counter.use_counter_page_window_cssvariablereferencevalue
+    ) AS use_counter_page_window_cssvariablereferencevalue,
+    SUM(
+      metrics.counter.use_counter_page_window_defaultstatus
+    ) AS use_counter_page_window_defaultstatus,
+    SUM(
+      metrics.counter.use_counter_page_window_devicemotioneventacceleration
+    ) AS use_counter_page_window_devicemotioneventacceleration,
+    SUM(
+      metrics.counter.use_counter_page_window_devicemotioneventrotationrate
+ 
```

Comment:
It's quite a long query, and my computer seems to really slow down trying to scroll through it. We calculate all these metrics and we only seem to use some of them at the end or did I miss something?